### PR TITLE
site: upgrade to next9 and react16 and style conventions

### PR DESCRIPTION
**Pull request by jaketrent**
asdf

### CODE REVIEW

File: packages/link/src/react/index.js [hunk 1]
```diff
@@ -1,31 +1,39 @@
...
-    ':hover': css['.psds-link:hover'],
-    ...(({ appearance }) => css[`.psds-link--appearance-${appearance}`])(props)
+  css(
+    {
+      ':hover': stylesheet['.psds-link:hover']
+    },
+    stylesheet[`.psds-link--appearance-${props.appearance}`]
+  )
+
+const Link = React.forwardRef((props, forwardedRef) => {
```
Comment (danethurber) [inline packages/link/src/react/index.js]: does this count as our react16 upgrade for Link? let's mark it off the list !

File: packages/site/package.json [hunk 2]
```diff
@@ -4,77 +4,66 @@
...
+    "@pluralsight/ps-design-system-textarea": "*",
+    "@pluralsight/ps-design-system-textinput": "*",
+    "@pluralsight/ps-design-system-theme": "*",
+    "@pluralsight/ps-design-system-tooltip": "*",
+    "@pluralsight/ps-design-system-util": "*",
+    "@pluralsight/ps-design-system-viewtoggle": "*",
+    "prop-types": "^15.7.2"
+  },
+  "dependencies": {
     "babel-preset-env": "^1.6.0",
```
Comment (danethurber) [inline packages/site/package.json]: do we still need these babel presets? we have removed our `.babelrc` file from the package root so i don't think that we'd be doing any custom babel config.

there is also a `next/babel` preset that is suggested in the docs but not sure what it does.

File: packages/site/pages/components/position-portal-example.js [hunk 3]
```diff
@@ -1,20 +1,17 @@
...
   const [node, setNode] = React.useState(portal.current)
-  React.useEffect(
-    () => {
-      if (portal.current) setNode(portal.current)
-    },
-    [portal]
-  )
+  React.useEffect(() => {
+    if (portal.current) setNode(portal.current)
+  }, [portal])
```
Comment (danethurber) [inline packages/site/pages/components/position-portal-example.js]: should this be `portal.current`. asking for a friend

File: packages/site/pages/components/table.js [hunk 4]
```diff
@@ -1,11 +1,11 @@
-import Avatar from '@pluralsight/ps-design-system-avatar/react'
+import Avatar from '@pluralsight/ps-design-system-avatar/react.js'
 import core from '@pluralsight/ps-design-system-core'
-import Drawer from '@pluralsight/ps-design-system-drawer/react'
+import Drawer from '@pluralsight/ps-design-system-drawer/react.js'
 import PT from 'prop-types'
```
Comment (danethurber) [inline packages/site/pages/components/table.js]: just a comment. maybe we should change our `PropTypes` component to be `PropTypesTable` or something. i just don't love the `PT` name. an issue for another day

File: packages/site/src/ui/index.js [hunk 5]
```diff
@@ -1,25 +1,49 @@
-export ChangeLog from './change-log'
```
Comment (danethurber) [inline packages/site/src/ui/index.js]: should we config babel to allow this type of export?

File: packages/link/src/react/index.js (hunk 1 above)
Comment (jaketrent) [reply packages/link/src/react/index.js]: yes, I'll mark it.

File: packages/site/package.json (hunk 2 above)
Comment (jaketrent) [reply packages/site/package.json]: babel-standalone uses plugins these inside the eval for code inside the live code component.  It's not for next per se.  It's used in the browser.

File: packages/site/pages/components/position-portal-example.js (hunk 3 above)
Comment (jaketrent) [reply packages/site/pages/components/position-portal-example.js]: I actually have no idea.  I just chose one and hoped for the best.  That's yeehaa-driven-development.

File: packages/site/pages/components/table.js (hunk 4 above)
Comment (jaketrent) [reply packages/site/pages/components/table.js]: Yeah, there's an odd naming clash there.  I don't love `PropTypes` either.  It's not what you anticipate it to be in the `site` package.

File: packages/site/src/ui/index.js (hunk 5 above)
Comment (jaketrent) [reply packages/site/src/ui/index.js]: I chose not to just for the sake of writing more "javascript today"-style code.  It also meant less configuration.

**Pull request merged**